.github/workflows/unit-tests.yaml

```
@@ -39,27 +39,19 @@ jobs:
-          sudo ln --symbolic --force /usr/bin/g++-10 /usr/bin/c++
-          sudo ln --symbolic --force /usr/bin/cpp-10 /usr/bin/cpp
+          brew install coreutils
+          brew install llvm@16
 
-      - name: "Log tool versions"
-        run: |-
-          command -v md5sum
-          command -v python
-          command -v tar
-          command -v task
+      - if: "'macos-14' == matrix.os"
```

actionlint (1.7.4)

step must run script with "run" section or run action with "uses" section

(syntax-check)